### PR TITLE
google credentials read from configuration

### PR DESCRIPTION
**Pull request by lpuskas**
* google credentials need to be provided as base64 encoded json in the configuration (formerly the path to the credentials json file was required)
* upgraded the google cloud client api version
* upgraded the opencensus version

fixes #234

### CODE REVIEW

File: pkg/cloudinfo/google/config.go [hunk 1]
```diff
@@ -15,6 +15,6 @@
 package google
 
 type Config struct {
-	ApiKey         string
-	AppCredentials string
+	ApiKey          string
```
Comment (sagikazarmark) [inline pkg/cloudinfo/google/config.go]: Is this still necessary then?

File: config.toml.dist [hunk 2]
```diff
@@ -40,8 +40,10 @@ prometheusQuery = "avg_over_time(aws_spot_current_price{region=\"%s\", product_d
 
 [provider.google]
 enabled = false
-apiKey = ""
-appCredentials = ""
+
+# bas64 encoded credentials in json format (base64 encoded content of the credential file)
```
Comment (sagikazarmark) [inline config.toml.dist]:
```suggestion
# base64 encoded credentials in json format (base64 encoded content of the credential file)
```

Comment (lpuskas) [reply config.toml.dist]: fixed, thx

File: pkg/cloudinfo/google/config.go (hunk 1 above)
Comment (lpuskas) [reply pkg/cloudinfo/google/config.go]: removed, thx

**Pull request merged**